packages/lab/src/list-next/useList.ts

```diff
@@ -1,54 +1,77 @@
 import {
+  FocusEvent,
   KeyboardEvent,
   RefObject,
   SyntheticEvent,
   useCallback,
+  useEffect,
   useMemo,
   useState,
 } from "react";
-import { useControlled } from "@salt-ds/core";
+import { useControlled, useIsFocusVisible } from "@salt-ds/core";
 
 interface UseListProps {
   /**
    * If true, all items in list will be disabled.
    */
   disabled?: boolean;
-  /* Value for the uncontrolled version. */
+  /* Value for the controlled version. */
```

Comment:
Could be better to mention what it controls. The word of "Controlled" can be multiple use cases.